pennylane/ops/op_math/sum.py

```diff
@@ -144,15 +143,26 @@ def __repr__(self):
     def __copy__(self):
         cls = self.__class__
         copied_op = cls.__new__(cls)
-        copied_op.data = self.data.copy()  # copies the combined parameters
         copied_op.summands = tuple(s.__copy__() for s in self.summands)
+        copied_op.data = self.data.copy()  # copies the combined parameters
```

Comment:
By copying the summands, the data should be copied as well. If not, there is a problem with the summands `__copy__` methods.

Reply:
yes! Thats weird, I think I removed that line at one point, but then re-added it for some reason.
```suggestion
```